src/common/i18n/translations/en.json

```diff
@@ -0,0 +1,362 @@
+{
+  "loginScanCard": {
+    "loginWithQR": "Please log in with your unique QR code provided by your supervisor",
+    "scanToLogin": "Scan to login",
+    "needHelp": "Need help?",
+    "askQuestion": "Ask a question"
+  },
+  "loginMobileNumberCard": {
+    "enterMobileNumber": "Please enter your mobile phone number to receive a one-time password.",
+    "mobileNumber": "Mobile phone number",
+    "sendOtp": "Send OTP"
+  },
+  "loginOTPCard": {
+    "sendingOtp": "We're sending you the one-time password...",
+    "Otp": "OTP",
+    "resendIn": "Resend in %{ss}s",
+    "submit": "Submit",
+    "resend": "Resend"
+  },
+  "collectCustomerDetailsScreen": {
+    "checkEligibleItems": "Check the number of item(s) eligible for redemption",
+    "scanIdentification": "Scan identification",
+    "or": "Or",
+    "enterIdNumber": "Enter ID Number",
+    "check": "Check"
+  },
+  "idScanner": {
+    "enterIdManually": "Enter ID manually",
+    "enterManually": "Enter manually",
+    "enterVoucherId": "Enter voucher ID",
+    "scanToCheck": "Scan to check validity"
+  },
+  "navigationDrawer": {
+    "logout": "Logout",
+    "addCampaign": "Add Campaign",
+    "selectCampaign": "Select campaign",
+    "changeChampaign": "Change Campaign",
+    "helpSupport": "Help & Support",
+    "termsOfUse": "Terms of use",
+    "privacyStatement": "Privacy Statement",
+    "reportVulnerability": "Report Vulnerability"
+  },
+  "customerQuotaScreen": {
+    "campaignExpiredOn": "Expired on %{dateTime}",
+    "campaignValidTo": "Valid till %{dateTime}",
+    "quotaRedeemedLimitReachedIDNumber": "Identification number(s)",
+    "quotaCategoryMealCredits": "Meal Credit(s)",
+    "quotaLimitMax": "Max",
+    "quotaAppealCancel": "Cancel",
+    "quotaButtonCheckout": "Checkout",
+    "quotaButtonAdd": "+ Add",
+    "quotaButtonAddVoucher": "Add voucher",
```

Comment:
Is this for merchant flow?

Reply:
Yep, the quotaButtonAddVoucher was inside the chinese translations gdoc so we added it in too (In case we need to use vouchers in the future)

Reply:
Hmm we might want to either move all merchant flow to a separate field or name it with merchant in the naming. So we are able to separate the two flows. Benefits would be new devs coming in will better understand why these copies don't appear in the app + if we ever clean up the merchant flow, it will be easier to find these bits and pieces. 

Personally I am leaning towards keeping the merchant stuff in a separate nested json like error messages.

What do you think?

Reply:
So something like 
```json 
...
"customerQuotaScreen": {
    "campaignExpiredOn": "Expired on %{dateTime}",
    "campaignValidTo": "Valid till %{dateTime}",
    "quotaRedeemedLimitReachedIDNumber": "Identification number(s)",
    "quotaLimitMax": "Max",
    "quotaAppealCancel": "Cancel",
    "quotaButtonCheckout": "Checkout",
    "quotaButtonAdd": "+ Add",
    "quotaAddId": "Add another ID to combine customer quotas",
    "quotaIdentifierButtonScan": "Scan",
    "quotaScanButtonBack": "Back",
  },
  merchantFlowScreen: {
    "quotaButtonAddVoucher": "Add voucher",
    "merchantCode": "Merchant Code"
  },
...
```
?

Reply:
Yup :)